bundles/org.eclipse.osgi/felix/src/org/apache/felix/resolver/Logger.java

```diff
@@ -22,7 +22,7 @@
 
 /**
  * <p>
- * This class mimics the standard OSGi <tt>LogService</tt> interface. An
+ * This class mimics the standard OSGi <code>LogService</code> interface. An
```

Comment:
I would prefer not to modify felix code for our coding standards.

Reply:
`<tt>` tag is deprecated by html5 instead either

- [kbd](https://wiki.selfhtml.org/wiki/HTML/Elemente/kbd)
- [var](https://wiki.selfhtml.org/wiki/HTML/Elemente/var)
- [code](https://wiki.selfhtml.org/wiki/HTML/Elemente/code)
- [samp](https://wiki.selfhtml.org/wiki/HTML/Elemente/samp)

must be used to this is not really a coding standard question and javadoc complains about it, I can remove it from this change as we do not generate javadoc for it but it probably should be addressed at felix then.